enterprise/app/controllers/enterprise/api/v1/accounts_controller.rb

```diff
@@ -1,4 +1,5 @@
 class Enterprise::Api::V1::AccountsController < Api::BaseController
+  include BillingHelper
```

Comment:
Let us add a before action on limits endpoint to see if it is cloud env, else send a 404.

Reply:
Done here: https://github.com/chatwoot/chatwoot/pull/7242/commits/9dc66618e703282feefd45165ed2a20e967e0dd5

Should I add this for subscription and checkout endpoint too?